docs/configuration/metrics/index.md

```diff
@@ -44,6 +44,7 @@ Every Azure service is supported and can be scraped by using the [Generic Azure
 
 We also provide a simplified way to configure the following Azure resources:
 - [Azure Service Bus Queue](service-bus-queue)
+- [Azure Queue](azure-queue)
```

Comment:
Please rename `Azure Queue` to `Azure Storage Queue` as Azure also provides Service Bus Queues